app/database_etl/postgres_tables_handler/postgres_loader.py

```diff
@@ -81,3 +81,9 @@ def drop_table_entries(current_time: datetime, drop_old: bool = True):
                 session.query(table).filter(or_(table.created_at == current_time, table.created_at.is_(None))).delete()
             session.commit()
     return
+
+def drop_specific_table_entries(current_time: datetime, table_name):
```

Comment:
are we gonna be using this function elsewhere? right now it just looks like we are using it in one place, and it looks like we are dropping all the records in the table anyways. i feel like we could just put the function logic where it's being called unless we are gonna use this again

Reply:
i think we will! this logic will be used for all filter options in the future (population_group was just the first test)

Reply:
nit: I would prefer this function being called `drop_old_table_entries`

Reply:
Also couldn't we just work this into the `drop_table_entries` function that we have currently, where we use the same pattern with all other tables??? I missed this earlier but tbh don't think we really need to have a whole new function/workflow here.

Reply:
good catch, just added `PopulationGroupOptions` to the `table_names_dict` so now it will be cleared with the `drop_table_entries` function!